### ISSUE TITLE
Outdated wheel version constraint

### ISSUE THREAD

**Issue by skwashd** [gorilla-co/s3pypi#95]
I have a project using s3pypi and some other packages for building and publishing releases. One of the packages depends on a newer version of wheel. I'm currently unable to upgrade.

The current stable version of s3pypi (0.11.1) depends on wheel 0.33.6. This version of wheel is over two and a half years old. Since then there have been 10 releases. Is it possible to support a more modern version of wheel, say 0.37.x?

**Comment by mdwint** [comment]
Hi @skwashd. Have you tried s3pypi 1.0.0rc3? It no longer depends on wheel. To get the same behaviour as 0.11.1, use:

```console
$ cd /path/to/your-project/
$ python setup.py sdist bdist_wheel
$ s3pypi dist/* --bucket your-bucket --unsafe-s3-website --acl public-read
```

The changelog explains these new CLI arguments in more detail.

If that doesn't work we could upgrade wheel on s3pypi 0.11.x, but I'd rather have people upgrade to 1.0.0.

**Comment by skwashd** [comment]
@mdwint Thanks for the fast reply.

I hadn't tried any of the 1.0.0 RCs. I was waiting for the stable release. 

I tried upgrading wheel and this is what I got:

```
$ poetry update
Updating dependencies
Resolving dependencies... (12684.2s)^C
```

I'll look at what it will take to port our tooling to 1.0.0rc3. Is there much still to be done before you cut a stable release?

**Comment by mdwint** [comment]
I've released 1.0.0 just now, along with some instructions for upgrading CloudFront etc. (#96). This is entirely optional though. Using `--unsafe-s3-website --acl public-read` should just work with your existing S3 bucket and CloudFront.

**Comment by skwashd** [comment]
Oh cool! I will take a look at it this week. I suspect we'll have a few changes to make in order to make this work.

Given 1.0.0 is stable, the fix here is clearly to upgrade. Thanks for your help. Closing.
